build/files.js

```diff
@@ -98,10 +98,47 @@ const headRegexp = /(^module.exports = \w+;?)/m
         + 'if (!EE.listenerCount) EE.listenerCount = function(emitter, type) {\n'
         + '  return emitter.listeners(type).length;\n'
         + '};\n/*</replacement>*/\n'
+      ]
+
+    , constReplacement = [
+        /const/g
+      , 'var'
+      ]
+
+    , bufferIsEncodingReplacement = [
+      /Buffer.isEncoding\((\w+)\)/
+    ,   '([\'hex\', \'utf8\', \'utf-8\', \'ascii\', \'binary\', \'base64\',\n'
+      + '\'ucs2\', \'ucs-2\',\'utf16le\', \'utf-16le\', \'raw\']\n'
+      + '.indexOf(($1 + \'\').toLowerCase()) > -1)'
+    ]
+
+    , requireStreamReplacement = [
+      /var Stream = require\('stream'\);/
```

Comment:
great pains are made to make sure `readableStream instanceof nativeStream` is true, this avoids bundling the streams module unless it's already included.  

In other words since you can't  `readableStream instanceof nativeStream` without first doing `var nativeStream = require('stream');` we only need to inherit from native streams if it's already bundled.